test/suite/test_util01.py

```diff
@@ -141,7 +145,24 @@ def dump_kv_to_line(self, b):
         # The output from dump is a 'u' format.
         return b.strip(b'\x00').decode() + '\n'
 
-    def dump(self, usingapi, hexoutput):
+    def write_entries(self, cursor, expectout, hexoutput, commit_timestamp, write_expected):
+        if commit_timestamp:
```

Comment:
Should be `if commit_timestamp is not None:`. I think `0` is not a truthy value and will evaluate false here.

Reply:
done!